### ISSUE TITLE
Training with multiple images of same person

### ISSUE THREAD

**Issue by ramaprakashganesan** [malikshubham827/get-me-through#8]
Probably more informational than an issue if there is already a way around this..

How can we add multiple images of the same person like X1.jpg, X2.jpg, X3.jpg, etc. in the known_people folder for more accurate training, and get to recognize the person as X?

**Comment by malikshubham827** [comment]
Well currently it needs only one photo to create the mapping, which is later used to calculate the distance from the captured image. It doesn't support multiple photos of same person. In future that can be possible, and an average of the mapping can be taken to find the person, but currently it didn't have that functionality!

**Comment by exexzian** [comment]
@malikshubham827 yes it would be much required one - to add multiple images of same person that it compares from multiple data-sets.
